src/Sanderling/Sanderling.Exe/sample/script/Mine.ore.cs

```diff
@@ -243,13 +243,27 @@ Func<object>	DefenseStep()
 	
 	var RatTargetNext = SetRatTarget?.OrderBy(target => target?.DistanceMax ?? int.MaxValue)?.FirstOrDefault();
 	
+	//Populating array to check if drones are fighting or not
+	Sanderling.Interface.MemoryStruct.IUIElementText[] droneArray = Sanderling.MemoryMeasurementParsed?.Value?.WindowDroneView?.FirstOrDefault()?.LabelText.ToArray();
```

Comment:
Well, i took the code as it was, i will check the `WindowDroneView?.ListView?.Entry?` var that you say and if it contains the required data i will refactor the code :)